catalyst_voices/packages/catalyst_voices_shared/lib/src/platform_widget_builder/platform_widget_builder.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:catalyst_voices_shared/src/platform/catalyst_platform.dart';
+import 'package:flutter/widgets.dart';
+
+class PlatformWidgetBuilder<T> extends StatelessWidget {
+  final Widget Function(BuildContext context, T? data) builder;
+  final T? android;
```

Comment:
I am just wondering why do we need these fields ?
During the widget `build` process we are totally relying on the `CatalystPlatform` data, or I am missing something ?

Reply:
The fields here are the platform-specific values we want to pass. The `CatalystPlatform` data is used just to determine the platform.

When using the widget with something like:

```
PlatformWidgetBuilder<String>(
    other: 'Other platform',
    iOS: 'iOS platform',
    builder: (context, title) => Text(title!),
);
```
we are displaying `iOS platform` by specifying it when we use the Widget. It will be visualized only when `CatalystPlatform` detects `iOS` and in all the other cases it will display `Other platform`.

Reply:
So am I correct that this class is it intended to provide additional data per specific platform ?

Reply:
Yes, that's correct!